lib/matplotlib/widgets.py

```diff
@@ -1473,6 +1473,7 @@ def new_axes(self, ax):
                                        transform=trans,
                                        visible=False,
                                        **self.rectprops)
+            self.stay_rect.set_animated(False)
```

Comment:
Trying to make sure I understand this bug (and make sure this is the right fix). Essentially, `self.rectprops` is going to have an "animated" entry that will be set to `True` when blitting is turned on. This, I think, is the essential bug that gets fixed in this line, because a static rectangle object isn't ever intended to be animated. So this part I am good with.